jekyll-assets/_layouts/docs.html

```diff
@@ -1,27 +1,187 @@
 <!doctype html>
 <html lang="en">
   <head>
-    {% include head.html %} 
+    {% include head.html %}
   </head>
-  <body>
-    {% include header.html %}      
-    {% include toptitle.html %}
-    {% include tabs.html %}
+  <body onload="makeToc()">
+    <script type="text/javascript">initTheme()</script>
+    <input type="checkbox" id="mobile-toggle" />
+    {% include header.html %}
+    <div id="docs-content">
+      <div id="toc-container-container">
+        <div id="toc-container">
+          <div id="toc-inner">
+            <div id="docs-header">
+              <h1 id="docs-header-title">
+                <a href="/documentation">
+                  Documentation
+                </a>
+              </h1>
+              <label class="mobile-menu-toggle" for="mobile-toggle"><div>≣</div></label>
+              {% include theme.html %}
+            </div>
+            <div id="docsearch">
+            </div>
+          </div>
+          <div class="toc">
+          {% for subdir in site.data.nav %}
+          {% if subdir.toc[0] %}
+            <ul class="sectlevel1">
+              <input class="toc-toggle-box" type="checkbox" id="{{ subdir.path }}"
+                {% if page.url contains subdir.path %}
+                checked=true
+                {% endif %}
+              />
+              <li>
+                <span>
+                  <label class="toc-toggle" for="{{ subdir.path }}" onclick="event.stopPropagation()">
```

Comment:
I have played around with this, but the label/checkbox pattern worked better for my purposes. details elements don't allow you to customise transitions.

Reply:
What transition are you trying to customise? The rotating disclosure arrow?

It looks like you're already rendering the arrow with a `::before` pseudo-element and then animating its transition with a sibling selector on `input:checked`. You should be able to do the same with `<details>` by [removing the default icon](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#customizing_the_disclosure_widget) and using `details[open]` to differentiate opened and closed elements.

Reply:
I'm not sure there's much of an advantage to using details here instead. For better or for worse, the label approach is what I'm more familiar with. Maybe we can address this in a follow-up PR later on, if we realise there's a significant advantage?

Reply:
Makes perfect sense, thanks for the explanation!

Reader mode seems to ignore the sidebar entirely (fair, since it isn't core content), but at screen widths where the on-this-page element displays, we do wind up with a big list of headers at the end of the reader mode document.

In a future update, I'll explore switching to `details` elements for the sidebar and also look into javascript-free alternatives to the current on-this-page widget. IMO it's OK for such a ancillary element to dispense accessibility, but preferably I'd find a better way that's more compatible with reader mode as well. Ultimately it would be best to eliminate tabs from the landing page and simplify the site rendering process, but I think we're iterating in the right direction for now. Just don't ask me about duplicate header names on the same (long) page -- there doesn't seem to be a great way to handle those right now.